client/src/components/Console.tsx

```diff
@@ -184,52 +185,86 @@ const Console: React.FC<ConsoleProps> = ({ user }) => {
+                                                <RiFileSettingsFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Technical Document</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiDraftFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Design Document</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiShakeHandsFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Agreement</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiFileSearchFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Consultation</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiFileCheckFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Conflict Resolution</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiFileChartFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Strategy</span>
+                                            </div>
+                                            <div className="flex items-center mb-2">
+                                                <RiFileFill className="mr-2" style={{ color: '#163C89' }} />
+                                                <span className="dark:text-white">Custom Document</span>
+                                            </div>
+                                        </div>
                                     </Card>
-                                    <Card className="hidden lg:block w-full h-40">
-                                        <img
-                                            src="/kiruna.png"
-                                            alt="Kiruna"
-                                            className="w-full h-full object-contain"
-                                        />
+                                    <Card className="p-4 h-1/2">
```

Comment:
<img width="1421" alt="Screen Shot 2024-12-18 at 20 03 42" src="https://github.com/user-attachments/assets/7407e469-186b-4e56-a431-dffce5b1f3d1" />

based on the time, it's not mandatory, but it would be better to move to two columns on small screens